src/Core/Picture2D.ts

```diff
@@ -245,4 +245,4 @@ interface pictureOptions {
      stretch?: boolean
 }
 
-export { Picture2D }
\ No newline at end of file
+export { Picture2D }
```

Comment:
hmm, not sure why github shows a change here. I did an edit in the file on github without touching this line. I'll see if I can revert this specific whitespace change (probably an OS related whitespace change, maybe a loss of a carriage return?)

Reply:
fixed via git client on machine. Seems a newline was added.